mslib/msui/qt5/ui_multiple_flightpath_dockwidget.py

```diff
@@ -114,9 +148,25 @@ def retranslateUi(self, MultipleViewWidget):
         _translate = QtCore.QCoreApplication.translate
         MultipleViewWidget.setWindowTitle(_translate("MultipleViewWidget", "Form"))
         self.ft_color_label.setText(_translate("MultipleViewWidget", "Activated Flighttrack/Operation Vertices Color: "))
+        self.cbSlectAll1.setText(_translate("MultipleViewWidget", "Select all list of Open Flighttracks"))
+        self.cbSlectAll2.setText(_translate("MultipleViewWidget", "Select all list of MsColab Flighttracks"))
         self.list_flighttrack.setToolTip(_translate("MultipleViewWidget", "List of Open Flighttracks.\n"
 "Check box to activate and display track on topview."))
         self.list_operation_track.setToolTip(_translate("MultipleViewWidget", "List of Mscolab Operations.\n"
 "Check box to activate and display track on topview."))
-        self.pushButton_color.setText(_translate("MultipleViewWidget", "Change Color"))
+        self.groupBox.setTitle(_translate("MultipleViewWidget", "Flight Track style options"))
+        self.pushButton_color.setText(_translate("MultipleViewWidget", "Change Color of flight track"))
+        self.label.setText(_translate("MultipleViewWidget", "Line thnickness"))
+        self.label_3.setText(_translate("MultipleViewWidget", "Line style"))
+        self.label_2.setText(_translate("MultipleViewWidget", "Transparency"))
         self.labelStatus.setText(_translate("MultipleViewWidget", "Status: "))
+
+
+if __name__ == "__main__":
```

Comment:
usually there is no main section, was this an option used?

Reply:
Ik I'll clean up